engines/bastion/app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -64,7 +64,14 @@ angular.module('Bastion.components').factory('Nutupane',
 
             params = params || {};
             params.paged = true;
-            params.page = $location.search().page || 1;
+
+            locationPage = $location.search().page;
+            // the $location page can return NaN as a String, explicitly checking for that here
+            if (!locationPage || isNaN(locationPage)) {
+              params.page = 1;
+            } else {
+              params.page = locationPage;
+            }
```

Comment:
This was necessary because sometimes the location is not paginated and will return "NaN" which somehow gets coerced into a string, so it becomes truthy and was not using the 1 as a default. This then throws off the Nutupane params and is also showing up in the URL params.

I added a more verbose check for a truthy value and `isNaN`, which checks for `"NaN"`